draft-irtf-cfrg-frost.md

```diff
@@ -812,6 +812,8 @@ Where Signature.R_encoded is `G.SerializeElement(R)` and Signature.z_encoded is
 The Coordinator SHOULD verify this signature using the group public key before publishing or
 releasing the signature. Signature verification is as specified for the corresponding
 ciphersuite; see {{ciphersuites}} for details.
+Recall that the Coordinator is configured with "group info" which contains
+the group public key `PK` and public keys `PK_i` for each participant.
```

Comment:
Can we move this down to L828? It doesn't seem useful here.

Reply:
Line 828? Done.